.github/scripts/setup_python_test_env.sh

```diff
@@ -6,12 +6,9 @@
 ### Install python packages
 ### ---------------------------------------------------------------------------
 
-echo "Install Python Packages"
+echo "Install packages"
 
-pip install -r ./requirements.txt
-pip install -r ./requirements-test.txt
-# -- need to upgrade urllib3 for GHA environment.
-pip install -U urllib3
+make init
```

Comment:
this is definitely a good start. i had so many issue with requirements.txt / setup.py. etc. great work!

Reply:
Thanks! My idea here is to try to reduce duplication as much as possible, but still keeping it easy to understand (hopefully). :-)